### ISSUE TITLE
editor sync on save

### ISSUE THREAD

**Issue by timokau** [tridactyl/tridactyl#1274]
When using the `editor` feature on websites with some sort of live preview or some other response to the entered text ([like this](https://crushentropy.com/plan), it would be nice to sync the text between the editor and the website from time to time.

One way to implement this would be to use inotify to detect changes on the temporary file and sync those changes to the input on the website. Another would be to have users set up an autocommand (or equivalent) to explicitly update tridactyl.

**Comment by timokau** [comment]
This would probably also be "fixed" by replacing `editor` with GhostText integration as proposed in #1267.

**Comment by bovine3dom** [comment]
Yeah, I'd rather not reimplement GhostText, but if anyone else does, I'd merge a PR that did.
